doc/user_manual/postprocessor.tex

```diff
@@ -166,6 +166,11 @@ \subsubsection{BasicStatistics}
     \end{itemize}
   \default{False}
   %
+  \item \xmlNode{multipleLinear}, \xmlDesc(boolean, optional field), this node can be used when the users do not want
+    to compute the multiple linear sensitivities. This method can be used when the input features depend on each
```

Comment:
This documentation should be extended and improved. For example,  "the users do not want to compute the multiple linear sensitivities"... what do we use instead? It is not explained and the user does not know.

Reply:
Also the name of the node is not self-explanatory

Reply:
updated, and I changed the name to ```multipleFeatures```